templates/templates.ts

```diff
@@ -331,5 +331,19 @@ export const templates: TemplateData[] = [
 		},
 		tags: ['chat'],
 		load: () => import('./open-webui/index').then((m) => m.generate),
+	},
+	{
+		id: 'doublezero',
+		name: 'Double Zero',
+		version: 'v0.0',
```

Comment:
Set the same version as in the service of the compose file

Reply:
they don't have yet a version name, that's why I set 0